cycledash/static/css/runs.css

```diff
@@ -43,6 +43,10 @@ td span.run-id {
   font-size: 0.92em;
 }
 
+.runs .run-id, .runs .date {
```

Comment:
Not really set on either way, but a simpler alternative is adding a `nowrap` (or, to be more semantic, `grouped-text`?) class, instead of a class for each column we don't want to wrap?